setup.sh

```diff
@@ -83,17 +83,16 @@ install_jdk() {
   fi
 }
 
-install_node() {
+install_nvm() {
   if ! installed node || ! installed npm; then
     if ! installed curl; then
       sudo apt-get install -y curl
     fi
 
     if linux; then
-      curl -sL https://deb.nodesource.com/setup | sudo bash -
-      sudo apt-get install -y nodejs
+      curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.2/install.sh | bash
     elif mac && installed brew; then
-      brew install node
+      brew install nvm
```

Comment:
i don't think it's too bad since nvm makes switching version super easy, as long as we're explicit that it's happening. people are gonna need to run that eventually for nvmrc to be useful anyway. 

not too fussed though, i guess without it we should mention that you need to do `nvm use` manually at some point for anyone not familiar with nvm